docs/source/index.rst

```diff
@@ -0,0 +1,81 @@
+.. cleanvision documentation master file, created by
+   sphinx-quickstart on Thu Feb  2 13:55:53 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+.. image:: https://raw.githubusercontent.com/cleanlab/assets/master/cleanlab/cleanvision_logo_open_source_transparent.png
+  :width: 800
+  :alt: CleanVision
+
+Documentation
+=======================================
+CleanVision automatically detects various issues in image datasets, such as images that are: (near) duplicates, blurry,
+over/under-exposed, etc. This data-centric AI package is designed as a quick first step for any computer vision project
+to find problems in your dataset, which you may want to address before applying machine learning.
+
+
+Installation
+===========
+
+To install cleanvision using pip:
+
+.. code-block:: console
+
+   $ pip install cleanvision
```

Comment:
can we use `shell` instead and remove the $ sign. `(.venv) $ pip install cleanvision` this gets copied otherwise

Reply:
This won't be copied after this change:
https://github.com/cleanlab/cleanvision/pull/84#discussion_r1103153682

Reply:
`console` syntactically highlights $ in a nicer way, is there a reason for choosing `shell`?

Reply:
shell looks better when there's no $. Since the regex handles the removal of $ while copying, either of them works.